.github/workflows/syncer.yml

```diff
@@ -0,0 +1,32 @@
+name: Grafana Syncer
+defaults:
+  run:
+    shell: bash
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Compile dashboards
+      run: make build
+
+    # The secret GRAFANA_URL must be set with the format http://username:password@url.org without a trailing /
+    - name: Import dashboards to grafana
+      run: >
+        t=("rendered/kube-burner.json");
```

Comment:
Only the kube-burner dashboard?
If so I think we should create a variable for that rather than hardcoding the dashboard name here.

Reply:
I updated it to follow this convention.